sktime/classification/deep_learning/rnn.py

```diff
@@ -51,7 +51,8 @@ class SimpleRNNClassifier(BaseDeepClassifier):
 
     def __init__(
         self,
-        num_epochs=100,
+        num_epochs = None,
+        n_epochs=100,
```

Comment:
n_epochs should be at the end, or it will break code for args users.
The warning message should state that a rename will happen.